src/sfdxCommand.ts

```diff
@@ -283,21 +279,20 @@ export abstract class SfdxCommand extends Command {
 
     // If the -h flag is set in argv and not overridden by the subclass, emit help and exit.
     if (this.shouldEmitHelp()) {
-      // eslint-disable-next-line no-underscore-dangle
-      this._help();
+      const Help = await loadHelpClass(this.config);
```

Comment:
If I understand correctly, this block is just to show help anytime `-h` is present? If that's the case, we should use this property in the oclif config instead: https://github.com/salesforcecli/cli/blob/main/package.json#L83

Although now that I'm thinking about it, we might still need this block since the rest of sfdx won't be on oclif/core yet

Reply:
yeah, I think this essentially is setting the `"additionalHelpFlags": ["-h"],` OCLIF setting for each extending command